### PR TITLE
AAP-29499: Added on-prem troubleshooting cross-reference and removed on-prem monitoring content

### PR DESCRIPTION
**Pull request by sayjadha**
This PR addresses JIRA https://issues.redhat.com/browse/AAP-29499.

Changes made:

- Reverted changes made in PR https://github.com/ansible/aap-docs/pull/1707. Some additional steps need to be added to the procedure, and because the PR is merged, this ticket reverses that change and starts afresh.
- Added hyperlinks to the 'troubleshooting on-prem deployment issues' topic from 'Setting up Lightspeed on-premise deployment'

### CODE REVIEW

File: lightspeed/modules/proc_create-connection-secrets.adoc [hunk 1]
```diff
@@ -139,4 +139,8 @@ secretGenerator:
       disableNameSuffixHash: true
 ....
 
-After you create the bundle secret, you must select the secret when you  xref:create-lightspeed-instance_configuring-lightspeed-onpremise[create and deploy an {LightspeedshortName} instance].
\ No newline at end of file
+After you create the bundle secret, you must select the secret when you  xref:create-lightspeed-instance_configuring-lightspeed-onpremise[create and deploy an {LightspeedshortName} instance].
```
Comment (jself-sudoku) [inline lightspeed/modules/proc_create-connection-secrets.adoc:142]: Minor nit; remove extra space.
```suggestion
After you create the bundle secret, you must select the secret when you xref:create-lightspeed-instance_configuring-lightspeed-onpremise[create and deploy an {LightspeedshortName} instance].
```

**Comment by jself-sudoku** [comment]
One tiny nit, which can ignore if you wish. Otherwise, LGTM.

File: lightspeed/modules/proc_create-connection-secrets.adoc (hunk 1 above)
Comment (sayjadha) [reply lightspeed/modules/proc_create-connection-secrets.adoc]: Thanks for catching that. Updated the doc

**Comment by sayjadha** [comment]
I've incorporated your feedback, @jself-sudoku . Request re-review before merging the PR.

**Pull request merged**